src/Client/app/(root)/my-exams/[id]/page.tsx

```
@@ -420,18 +453,18 @@
                                     timeLeft !== undefined && timeLeft < 300
-                                        ? "text-danger"
-                                        : "text-success"
+                                        ? "text-danger font-bold"
+                                        : "text-success font-bold"
                                 }`}
                             >
                                 {timeLeft !== undefined
                                     ? FormatTimeHourMinutesSeconds({
                                           seconds: timeLeft,
-                                      }) + " s"
+                                      }) + "s"
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)